apps/web/src/app/(private)/datasets/generate/GenerateDatasetContent.tsx

```diff
@@ -132,6 +141,15 @@ export function GenerateDatasetContent() {
     })
   }
 
+  useEffect(() => {
+    if (defaultName && defaultParameters && runPreviewAction) {
+      runPreviewAction({
+        parameters: defaultParameters.join(','),
+        description: defaultName,
+      })
+    }
+  }, [defaultName, defaultParameters])
```

Comment:
Missing `runPreviewAction` we need to bring back react hooks rules

Reply:
pues si